site/content/3.12/aql/functions/date.md

```diff
@@ -401,34 +528,68 @@ description: ''
 RETURN DATE_MILLISECOND("2020-08-29T16:30:05.123")
 ```
 
+```aql
+---
+name: datesec2
```

Comment:
The duplicate example name resulted in this example overwriting the previous one for DATE_SECOND(), apparently without a warning/error!
```suggestion
name: datemilsec2
```

Reply:
Hmm. Let's update its name and re-run the example generation. I will create a ticket to fix this.